### ISSUE TITLE
Platform is not being used properly?

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/universal-xapi-translator#5]
“UxAPI” appears to be a generic placeholder and not reflective of the platform which is the source. This should be the simple name of the D&G attendance system.

**Comment by michaelwebbjisc** [comment]
Added a new xxxx.meta file to the process (eg attendance.meta) that gives a platform name like this:
{
	"system":"D&G Attendance"
}

**Comment by arc12** [comment]
presume this means the xAPI statement gets a simple string for "platform", as per the xAPI spec

**Comment by michaelwebbjisc** [comment]
Yes, although it's a bit of a stretch for attendance tbh.  The spec says 'Platform used in the experience of this learning activity'.  I was interpreting it as 'The system writing the data', whereas I think we want 'The system that is providing the data'.
